### DISCUSSION TITLE
How do I install all star catalogs in the stellarium-web-engine?

### DISCUSSION THREAD

**Discussion by G-jd0726** [Stellarium/stellarium#3852] [Q&A]
Hi Folks,

I have built the stellarium-web-engine in my local environment, but I only have a little test data in test-skydata. How can I install all the sky data?

**Comment by gzotti** [comment]
Could be a question to the project in question, stellarium-web. @xalioth ?

**Reply by G-jd0726** [reply]
Yes, the question is about the Stellarium-web project.

**Comment by obodak** [comment]
Yes, I agree
